README.md

```diff
@@ -26,6 +26,25 @@ On Windows, there are binary wheel you can install without MySQLConnector/C or M
 `sudo yum install python3-devel `  # Red Hat / CentOS
 
 `brew install mysql-connector-c`  # macOS (Homebrew)
+#### MySQL note
+Versions of `mysql`may have incorrect default configuration options that cause compilation errors when `mysqlclient-python` is installed.  (As of November 2017, this is known to be true for homebrew's `mysql-connector-c` and `mysql` packages) 
```

Comment:
Replace "mysql" with MySQL Connector/C.

When you did `brew install mysql`, it is OK.
When you did `brew install mysql-connector-c`, `mysql_config` is broken.

Reply:
Add link to https://bugs.mysql.com/bug.php?id=86971